packages/styles/scss/components/masthead/_masthead-megamenu.scss

```
@@ -123,13 +132,109 @@
       @include carbon--make-container;
 
       width: 100vw;
+    }
+
+    .#{$prefix}--masthead__megamenu__container--row {
+      background-color: $ui-03;
 
-      .#{$prefix}--masthead__megamenu__container--row {
+      /**
+       * Faceted (tab) and Simple (list) megamenus have different layouts between
+       * 'md' and 'lg' breakpoints. This requires us to apply identical styles at
+       * different breakpoints, and we must also leverage custom properties to assign
+       * values across shadow roots. The concept is that these upper shadow roots have
+       * the stateful information needed to dictate how elements in the lower shadow
+       * roots should appear, and we can pass that information down via custom props.
+       */
+      &.#{$prefix}--masthead__megamenu__container--row--tab {
         @include carbon--make-row;
 
-        background-color: $ui-03;
         flex-wrap: nowrap;
         gap: 1px;
+
+        ::slotted(#{$dds-prefix}-megamenu-right-navigation) {
+          --#{$dds-prefix}-right-nav-padding: #{$layout-01};
+        }
+      }
+
+      &.#{$prefix}--masthead__megamenu__container--row--list {
+        @include carbon--breakpoint-down(lg) {
+          @include masthead-megamenu-limit-height;
+          --#{$dds-prefix}-view-all-inset: calc(#{$layout-01} * -1);
+          --#{$dds-prefix}-view-all-container-position: initial;
+          --#{$dds-prefix}-view-all-container-transform: initial;
+
+          // Set containing block for fixed position view all CTA.
+          transform: rotate(0deg);
+          position: relative;
+          z-index: 1;
+
+          margin-inline: calc(#{$layout-04} * -1);
+        }
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-row;
+
+          flex-wrap: nowrap;
+          gap: 1px;
+        }
+
+        ::slotted(#{$dds-prefix}-megamenu-left-navigation),
+        ::slotted(#{$dds-prefix}-megamenu-right-navigation) {
+          @include carbon--breakpoint-down(lg) {
+            --#{$dds-prefix}-view-all-padding: #{$layout-01};
+            --#{$dds-prefix}-masthead-max-height: initial;
+            --#{$dds-prefix}-masthead-overflow: initial;
+
+            width: 100%;
+            max-width: initial;
+            margin-inline: initial;
+            padding-inline: $layout-03;
+            border-bottom: 1px solid $ui-03;
```

```suggestion
            border-bottom: carbon--rem(1px) solid $ui-03;
```

Nitpicking: Let's keep this to `rem` and consistent with how it's being used here (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10190/files#diff-56ec093065bb965a14aa4a46ec7c41d439780a22109f50f4ebed646596f10fb3R287)